api/features/views.py

```diff
@@ -154,8 +154,26 @@ def get_queryset(self):
         )
         queryset = queryset.order_by(sort)
 
+        if environment_id:
+            page = self.paginate_queryset(queryset)
+
+            environment = Environment.objects.get(id=environment_id)
+            feature_states = FeatureState.objects.get_live_feature_states(
+                environment,
+                additional_filters=Q(feature_id__in=[feature.id for feature in page]),
```

Comment:
I think we also need to filter out segment overrides and identity overrides here, right?

Reply:
Oh! I was under the wrong impression that the `get_live_feature_states` method handled more than it does. I'll add in the filters for the segment overrides and identity overrides here and in the other PR.